storage/tokudb/tokudb_thread.h

```diff
@@ -204,21 +204,21 @@ inline mutex_t::mutex_t(pfs_key_t key) {
     _owners = 0;
     _owner = _null_owner;
 #endif
-    int r = mysql_mutex_init(key, &_mutex, MY_MUTEX_INIT_FAST);
+    int  r __attribute__((__unused__)) = mysql_mutex_init(key, &_mutex, MY_MUTEX_INIT_FAST);
```

Comment:
```s/__attribute__/MY_ATTRIBUTE/g```
@georgelorchpercona , I am quite certain there was legacy/current JIRA issue re. this compilation warning, but I am unable to find it?

Reply:
I've used ```__attribute__``` because it's in our own code, and in it's repository, tokudb uses this syntax.

So in the mysql repo, even for our own code, I should use the mysql conventions?

Reply:
Yes, unless we have reasons to do otherwise. It's still MySQL code.